### PR TITLE
Develop   delete user fix

### PR DESCRIPTION
**Pull request by syphzur**

### CODE REVIEW

**Review by latusikl — CHANGES_REQUESTED** [review]
Clear Address. Change CannotDeleteException to RuntimeException. Reformat and optimize imports.

File: PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java [hunk 1]
```diff
@@ -60,6 +61,10 @@
     @Patchable
     private UserRole userRole;
 
+    @Column(name = "is_enabled")
+    @ColumnDefault("TRUE")
```
Comment (latusikl) [inline PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java:65]: This should be good for updating current users. But I'm not sure if it works. Have You checked that? In db boolean is represented as 0 or 1. I don't think that it works if you have "TRUE". I does not work when you run app from docker-compose. While creating user value in db is 0 and you cannot log in.

File: PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java [hunk 2]
```diff
@@ -60,6 +61,10 @@
...
+    private boolean isEnabled;
```
Comment (latusikl) [inline PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java]: Please add:
```java
private boolean isEnabled = true;
```

**Review by latusikl — CHANGES_REQUESTED** [review]

File: PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java (hunk 1 above)
Comment (syphzur) [reply PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java:65]: column generated by hibernate has type bit(1) with default value 1 so it works

**Review by latusikl — APPROVED** [review]

**Pull request merged**